src/main/java/uk/gov/hmcts/divorce/document/content/ConditionalOrderOfflineClarificationContent.java

```diff
@@ -0,0 +1,122 @@
+    public static final String REASON_MARRIAGE_CERTIFICATE = "marriageCertificate";
+    public static final String REASON_PREVIOUS_PROCEEDINGS_DETAILS = "previousProceedingDetails";
+
+    @Autowired
+    private Clock clock;
+
+    @Autowired
+    private ConditionalOrderRefusalContent conditionalOrderRefusalContent;
+
+    @Value("${court.locations.serviceCentre.serviceCentreName}")
+    private String serviceCentre;
+
+    @Value("${court.locations.serviceCentre.centreName}")
+    private String centreName;
+
+    @Value("${court.locations.serviceCentre.poBox}")
+    private String poBox;
+
+    @Value("${court.locations.serviceCentre.town}")
+    private String town;
+
+    @Value("${court.locations.serviceCentre.postCode}")
+    private String postcode;
+
+    @Value("${court.locations.serviceCentre.phoneNumber}")
+    private String phoneNumber;
+
+    public Map<String, Object> apply(final CaseData caseData, final Long ccdCaseReference) {
+
+        Map<String, Object> templateContent = new HashMap<>();
+
+        final ConditionalOrder conditionalOrder = caseData.getConditionalOrder();
+        final Set<ClarificationReason> clarificationReasons = conditionalOrder.getRefusalClarificationReason();
+
+        templateContent.put(CCD_CASE_REFERENCE, formatId(ccdCaseReference));
+        templateContent.put(DATE, LocalDate.now(clock).format(DATE_TIME_FORMATTER));
+
+        templateContent.put(APPLICANT_1_FULL_NAME, caseData.getApplicant1().getFullName());
+        templateContent.put(APPLICANT_2_FULL_NAME, caseData.getApplicant2().getFullName());
+
+        templateContent.put(IS_SOLE, caseData.getApplicationType().isSole());
+        templateContent.put(IS_JOINT, !caseData.getApplicationType().isSole());
+
+        templateContent.put(JUDICIAL_SEPARATION,
+            caseData.getIsJudicialSeparation() != null && caseData.getIsJudicialSeparation().toBoolean());
+
+        templateContent.put(REASON_JURISDICTION_DETAILS,
+            clarificationReasons.contains(ClarificationReason.JURISDICTION_DETAILS));
+        templateContent.put(REASON_MARRIAGE_CERT_TRANSLATION,
+            clarificationReasons.contains(ClarificationReason.MARRIAGE_CERTIFICATE_TRANSLATION));
+        templateContent.put(REASON_MARRIAGE_CERTIFICATE,
+            clarificationReasons.contains(ClarificationReason.MARRIAGE_CERTIFICATE));
+        templateContent.put(REASON_PREVIOUS_PROCEEDINGS_DETAILS,
+            clarificationReasons.contains(ClarificationReason.PREVIOUS_PROCEEDINGS_DETAILS));
+
+        templateContent.put(LEGAL_ADVISOR_COMMENTS, conditionalOrderRefusalContent.generateLegalAdvisorComments(conditionalOrder));
+
+        if (caseData.getDivorceOrDissolution().isDivorce()) {
+            templateContent.put(MARRIAGE_OR_CIVIL_PARTNERSHIP, MARRIAGE);
+            templateContent.put(DIVORCE_OR_CIVIL_PARTNERSHIP, DIVORCE);
+            templateContent.put(DIVORCE_OR_CIVIL_PARTNERSHIP_EMAIL, CONTACT_DIVORCE_JUSTICE_GOV_UK);
```

Comment:
placeholders DIVORCE_OR_CIVIL_PARTNERSHIP and DIVORCE_OR_CIVIL_PARTNERSHIP_EMAIL are not used in the template, either for divorce or cp, so I think we can remove them.

Reply:
Thanks Sunil for the review. I've removed these and taken care of your other comment 👍